### PR TITLE
Fix isAnimationRunning()

### PR DESCRIPTION
**Pull request by mwgriffiths88**

### CODE REVIEW

File: lib/progress_button.dart [hunk 1]
```diff
@@ -104,7 +104,7 @@ class _ProgressButtonState extends State<ProgressButton>
 
   /// A utility function to check whether an animation is running
   bool isAnimationRunning(AnimationController controller) {
-    return !(controller.isCompleted || controller.isDismissed);
+    return controller.isCompleted && controller.isDismissed;
```
Comment (halilozercan) [inline lib/progress_button.dart]: Just reading through the code, this doesn't seem right to me. This says in plain english;
> If animation controller is completed and is dismissed at the same time, then the animation is running.

Comment (mwgriffiths88) [reply lib/progress_button.dart]: It's true. As you say that I realise that I've changed the focus of the method to check if the animation is complete, rather than whether it's running. I've added a new commit that changes this to:

If animation controller is not completed, and is not dismissed, then the animation is running.

**Comment by iamdiogo** [comment]
Anything missing? Seems about right @halilozercan

**Review by therealblue10 — APPROVED** [review]
All good, let's merge it
